src/smartAuthorizationHelper.ts

```diff
@@ -138,8 +153,9 @@ export function hasAccessToResource(
     return (
         hasSystemAccess(usableScopes, sourceResource.resourceType) ||
         (fhirUserObject &&
-            (isFhirUserAdmin(fhirUserObject, adminAccessTypes, apiUrl) ||
```

Comment:
Did you intentionally remove the isFhirUserAdmin condition?  Shouldn't we keep it in?

Reply:
yes, I removed the isFhirUserAdmin condition as FhirUser is no longer Admin now.

Reply:
Weren't your changes on lines 81,82 of smartHandler.ts enough to remove admin user access to the sytem?

Reply:
Hmm, `isFhirUserAdmin` returning false would be expected in this case, and it would mean that another part of the 'OR' condition would need to return true. I think it is probably fine to omit it right now as we aren't expecting there to be an 'admin' user. Those functions would be reserved for system-level access. 

It's hard to tell, but I think there is still an extra check on fhirUserObject. Shouldn't it be like this?
```
hasSystemAccess(...) ||
(fhirUserObject && hasReferenceToResource(fhirUserObject, ...) ||
(patientOrgs && hasReferenceToResource(patientOrgs, ...)) ||
(patientLaunchContext && hasReferenceToResource(patientLaunchContext, ... ))
```

Reply:
we would still require fhirUserObject which consist of the practitioner details. So fhirUserObject check condition is required when patientOrgs is present.

Reply:
But in the patientOrgs conditional we don't require fhirUserObject to pass into the call to hasReferenceToResource. It is a slight optimization since fhirUserObject will always be set when patientOrgs is set.

Reply:
Okay, Sounds good to me.